### ISSUE TITLE
Triggered by head trauma

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4808]
**Preferred term label:**

**Synonyms**

Head trauma triggers worsening
**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0031135

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by MickeySegal** [comment]
Vanishing white matter disease is a good example of a disease needing the head-specific finding.
